homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/gravatar/components/AvatarsGridSection.kt

```diff
@@ -64,18 +52,72 @@ internal fun LazyGridScope.avatarsGridSection(
             )
         }
     } else {
+        if (avatars.none { it is AvatarUi.Uploaded && it.isSelected }) {
+            item(
+                span = { GridItemSpan(maxLineSpan) },
+            ) {
+                NoAvatarSelectedBanner(
+                    modifier = Modifier
+                        .animateItem()
+                        .fillMaxWidth()
+                        .padding(top = 10.dp)
+                )
+            }
+        }
         items(items = avatars, key = { it.avatarId }) { avatarModel ->
             SelectableAvatar(
                 avatar = avatarModel,
                 size = avatarSize,
-                modifier = Modifier.animateItem(),
+                modifier = Modifier
+                    .animateItem()
+                    .padding(top = 10.dp),
```

Comment:
❓ Did you add this padding on purpose? It's not in the designs.

Reply:
🤦 No, this should be elsewhere. Thanks for spotting this!